docs/dell.md

```diff
@@ -0,0 +1,135 @@
+
+Iceberg can be used with Dell's Enterprise Object Storage (ECS) by using the ECS catalog since 0.15.0.
+
+See [Dell ECS](https://www.dell.com/en-us/dt/storage/ecs/index.htm) for more information on Dell ECS.
+
+### Parameters
+
+When using Dell ECS with Iceberg, these configuration parameters are required:
+
+| Name                     | Description                       |
+| ------------------------ | --------------------------------- |
+| ecs.s3.endpoint          | ECS S3 service endpoint           |
+| ecs.s3.access-key-id     | ECS Username                      |
+| ecs.s3.secret-access-key | S3 Secret Key                     |
+| warehouse                | The location of data and metadata |
+
+The warehouse should use the following formats:
+
+| Example                    | Description                                                     |
+| -------------------------- | --------------------------------------------------------------- |
+| ecs://bucket-a             | Use the whole bucket as the data                                |
+| ecs://bucket-a/            | Use the whole bucket as the data. The last `/` is ignored.      |
+| ecs://bucket-a/namespace-a | Use a prefix to access the data only in this specific namespace |
+
+The Iceberg `runtime` jar supports different versions of Spark and Flink. You should pick the correct version.
+
+Even though the [Dell ECS client](https://github.com/EMCECS/ecs-object-client-java) jar is backward compatible, Dell EMC still recommends using the latest version of the client.
+
+### Spark
+
+To use the Dell ECS catalog with Spark 3.2.1, you should create a Spark session like:
+
+```bash
+ICEBERG_VERSION=0.15.0
+SPARK_VERSION=3.2_2.12
+ECS_CLIENT_VERSION=3.3.2
+
+DEPENDENCIES="org.apache.iceberg:iceberg-spark-runtime-${SPARK_VERSION}:${ICEBERG_VERSION},\
+org.apache.iceberg:iceberg-dell:${ICEBERG_VERSION},\
+com.emc.ecs:object-client-bundle:${ECS_CLIENT_VERSION}"
+
+spark-sql --packages ${DEPENDENCIES} \
+    --conf spark.sql.extensions=org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions \
+    --conf spark.sql.catalog.my_catalog=org.apache.iceberg.spark.SparkCatalog \
+    --conf spark.sql.catalog.my_catalog.warehouse=ecs://bucket-a/namespace-a \
+    --conf spark.sql.catalog.my_catalog.catalog-impl=org.apache.iceberg.dell.ecs.EcsCatalog \
+    --conf spark.sql.catalog.my_catalog.ecs.s3.endpoint=http://10.x.x.x:9020 \
+    --conf spark.sql.catalog.my_catalog.ecs.s3.access-key-id=<Your-ecs-s3-access-key> \
+    --conf spark.sql.catalog.my_catalog.ecs.s3.secret-access-key=<Your-ecs-s3-secret-access-key>
+```
+
+Then, use `my_catalog` to access the data in ECS. You can use `SHOW NAMESPACES IN my_catalog` and `SHOW TABLES IN my_catalog` to fetch the namespaces and tables of the catalog.
+
+The related problems of catalog usage:
+
+1. The `pyspark.sql.SparkSession.catalog` won't access the 3rd-party catalog of Spark, so please use DDL SQL to list all tables and namespaces.
```

Comment:
Only the `pyspark.sql.SparkSession.catalog` has this problems ?  How about the java & scala ?

Reply:
All languages are the same. I have fixed this statement.